src/logs_syslog_unix.mli

```diff
@@ -26,6 +26,21 @@ val tcp_reporter : ?hostname:string -> Unix.inet_addr -> ?port:int ->
   ?framing:Logs_syslog.framing -> unit ->
   (Logs.reporter, string) result
 
+(** [unix_reporter ~socket ~truncate ~framing ()] is [Ok reporter] or
+    [Error msg]. The [reporter] sends each log message via syslog to [socket]
+    (which defaults to ["/dev/log"]). If the initial connection to the socket
+    fails, the log message is reported to standard error, and attempts are made
+    to re-establish the connection. A syslog message is truncated to [truncate]
+    bytes and is framed according to the given [framing]. The default for
+    [truncate] is [65536] is [framing] is not provided and [0] otherwise. If
```

Comment:
typo: second `is` in line 36 should be `if`

Reply:
Oops, ta!